[2022-11-29T03:57:20.329+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: data_generator_dag.data_generation scheduled__2022-11-28T00:00:00+00:00 [queued]>
[2022-11-29T03:57:20.337+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: data_generator_dag.data_generation scheduled__2022-11-28T00:00:00+00:00 [queued]>
[2022-11-29T03:57:20.338+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T03:57:20.339+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-29T03:57:20.339+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T03:57:20.382+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): data_generation> on 2022-11-28 00:00:00+00:00
[2022-11-29T03:57:20.403+0000] {standard_task_runner.py:55} INFO - Started process 64 to run task
[2022-11-29T03:57:20.434+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generator_dag', 'data_generation', 'scheduled__2022-11-28T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/dag_generate_data.py', '--cfg-path', '/tmp/tmpyolhfqik']
[2022-11-29T03:57:20.446+0000] {standard_task_runner.py:83} INFO - Job 19: Subtask data_generation
[2022-11-29T03:57:20.553+0000] {task_command.py:376} INFO - Running <TaskInstance: data_generator_dag.data_generation scheduled__2022-11-28T00:00:00+00:00 [running]> on host 18e45f209dfb
[2022-11-29T03:57:20.641+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generator_dag
AIRFLOW_CTX_TASK_ID=data_generation
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T00:00:00+00:00
[2022-11-29T03:57:20.643+0000] {logging_mixin.py:137} INFO - current spot /opt/***
[2022-11-29T03:57:20.644+0000] {logging_mixin.py:137} INFO - current files ['dags', 'logs', '***-worker.pid', 'plugins', '***.cfg', 'webserver_config.py', '{PATH_PREICT}predictions.csv']
[2022-11-29T03:57:20.682+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-29T03:57:20.690+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=data_generator_dag, task_id=data_generation, execution_date=20221128T000000, start_date=20221129T035720, end_date=20221129T035720
[2022-11-29T03:57:20.718+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-29T03:57:20.734+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
